compiler/luci/pass/src/SubstituteTransposeToReshapePass.test.cpp

```diff
@@ -105,8 +91,15 @@ TEST_F(SubstituteTransposeToReshapeTest, simple_case)
   ASSERT_EQ(201, new_shape->at<loco::DataType::S32>(3));
 }
 
-TEST_F(SubstituteTransposeToReshapeTest_NEG, simple_case_NEG)
+TEST_F(SubstituteTransposeToReshapeTest, simple_case_NEG)
```

Comment:
how is this test case negative?

Reply:
um...I think this is a case that will not make any change on. so, I  named as neg.

Reply:
> I think this is a case that will not make any change on

Then it would be better to add a comment about, something like what we expect but actually doesn't work like that...

Reply:
@seanshpark Umm....little bit confused. on this [comment](https://github.com/Samsung/ONE/pull/5457#issuecomment-752308952), the case that not meet activation condition is negative test. I think this case is also not meet condition intentionally.

Reply:
> how is this test case negative?

This was a question about how this was a negative test. I wanted to know your idea. But instead of answering, you've removed the `_NEG`. I am also confused about this. I still hope to there is a negative test.

Reply:
> how is this test case negative?

@seanshpark Sorry for confusing and misunderstanding. I just thought that the question means that the case is not negative. I will update that case as `NEG`, as that case is intentionally make a condition for not changing.